FrontEnd/src/components/basics/Alert/Alert.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FunctionComponent } from 'react';
+import { Icon as IconType, alertIcons } from '@icons';
+import styles from './Alert.module.scss';
+import Button from '@components/basics/Button';
+import cn from 'classnames';
+
+export type alertVariant = 'success' | 'alert' | 'warning' | 'info';
+
+interface AlertProps {
+  variant: alertVariant;
+  title?: string;
+  text: string;
+  button?: string;
+}
+
+function getIcon(variant: string): string {
```

Comment:
it is recommended to use an object: e.g. const IconMapping = {'success':alertIcons.Check, ..... }

Reply:
fixed